rewriter/Prop.cc

```diff
@@ -233,6 +233,11 @@ optional<PropInfo> parseProp(core::MutableContext ctx, const ast::Send *send) {
         ENFORCE(ctx.locAt(sym->loc).exists());
         ENFORCE(!ctx.locAt(sym->loc).source(ctx).value().empty() && ctx.locAt(sym->loc).source(ctx).value()[0] == ':');
         ret.nameLoc = core::LocOffsets{sym->loc.beginPos() + 1, sym->loc.endPos()};
+        const auto nameValue = ctx.locAt(ret.nameLoc).source(ctx).value();
```

Comment:
this is a bit disgusting, but just fixes the loc to ignore the quotes for the error message

Happy to extract the quoting conditional into a separate function or something, not sure what might be a good place for it though